src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0], when ages inputs are zero', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0], when ages inputs are less then 15', () => {
+    expect(getHumanAge(8, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0], when ages inputs are negative', () => {
+    expect(getHumanAge(-12, -100)).toEqual([0, 0]);
+  });
+
+  test('should rounded, when ages inputs are non-integer numbers', () => {
+    expect(getHumanAge(130.4, 1412.412)).toEqual([28, 279]);
+  });
```

Comment:
unnecessary checks, age can be only positive and a function accepts only integers. pay attention to the task description